### ISSUE TITLE
Failed to train 3d-gated model with FVI 

### ISSUE THREAD

**Issue by AllenAnthony** [amjltc295/Free-Form-Video-Inpainting#20]
Hi,
I used the config in config.json and FVI_all_masks.json to train a 3d gated model in FVI.
I modified the loss_gan_temporal_weight to 1.0 and pretrained the model for 3days without gan loss first.
The model has been trained more than 5 days on 4 GPUs with batch size 8 and has been convergent already. However, the performance is far away as good as the value in your paper.

So, How can we modify the config to obtain the performance as good as the value in your paper?
Are the losses' weights in my setting correct? (loss_recon: 1.0, loss_masked_recon: 1.0, loss_vgg: 1.0, loss_style: 10, loss_gan_spatial_weight: 1.0)

Thank you!

**Comment by amjltc295** [comment]
You should not set loss_gan_temporal_weight as 1.0 for pretrain. Set it 0.1 for fine-tuning.
loss_gan_temporal_weight and loss_gan_spatial_weight are both GAN losses and should be set to 0 for pretraining. 
loss_vgg 10, loss_style 120 may be better.
Other than that, you have to tune the parameters yourself as batch size are different.
